docs/tutorials/typescript/nextjs/index.md

```diff
@@ -1,290 +1,626 @@
+  "extends": "@tsconfig/node20/tsconfig.json",
   "version": "4.4.2",
   "compilerOptions": {
-    "emitDecoratorMetadata": false,
-    "experimentalDecorators": false,
     "declaration": true,
     "declarationMap": true,
     "sourceMap": true,
-    "composite": true,
     "rootDir": "./src",
     "outDir": "./lib"
   },
-  "include": ["src/**/*.ts"],
-  "exclude": ["node_modules"]
+  "include": ["src/**/*.ts"]
 }
 ```
 
-For convenience, you may want to set up some npm scripts to run the builds in your project root `package.json`.
+For convenience, set up some scripts to run the builds in your project root's `package.json`.
 
+Add the `npm-run-all` command to your project as a dependency:
 
-```js
-// /package.json
+```command
+npm install npm-run-all --save-dev
+```
+
+Then use `npm-run-all` to define scripts to run your Temporal build processes and run the Next.JS app alongside Temporal Workers.
+
+Locate your existing `scripts` section, which defines  scripts to run and build Next.js applications:
+
+```json
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+```
+
+Change the scripts so you can build and run your Next.js application and your Temporal Workers at the same time:
+
+```json
   "scripts": {
     "dev": "npm-run-all -l build:temporal --parallel dev:temporal dev:next start:worker",
     "dev:next": "next dev",
     "dev:temporal": "tsc --build --watch ./temporal/tsconfig.json",
     "build:next": "next build",
     "build:temporal": "tsc --build ./temporal/tsconfig.json",
-    "start": "npm run dev",
     "start:worker": "nodemon ./temporal/lib/worker",
+    "start": "next start",
     "lint": "eslint ."
   },
 ```
 
-In the above example we use `npm-run-all` and `nodemon` so that we are able to do 4 things:
+These scripts let you run the following steps with a single `npm run dev` command:
+
+- build Temporal once.
+- start Next.js locally.
+- start a Temporal Worker.
+- rebuild Temporal files when they change.
+
+With your project configured, you can write your Temporal Workflows and Activities.
 
-- build Temporal once
-- start Next.js locally
-- start a Temporal Worker
-- rebuild Temporal files on change
+## Define the business logic using Temporal
 
-in a single `npm run dev` command.
+You'll use Temporal to power a one-click ordering process.
+You'll use a Temporal Workflow to represent each order.
+That Workflow will orchestrates one or more Activities.
 
+[Workflows](https://docs.temporal.io/workflows) define the overall flow of your business process.
+Conceptually, a Workflow is a sequence of steps written in your programming language.
 
-## Write your first Workflow, Activity and Worker
+Workflows orchestrate [Activities](https://docs.temporal.io/activities), which is how you interact with the outside world in Temporal.
+You use Activities to make API requests, access the filesystem, or perform other non-deterministic operations.
```

Comment:
1. I would remove "Workflows orchestrate Activities" here or line 151, because you just mentioned "Workflows orchestrate Activities" very quickly after the first mention.
2. I would personally move lines 153-157 to above 150-151. This is because you mention that the Workflow orchestrates the Activities before the user knows what a Workflow/Activity is specifically. So like this:

[Workflows](https://docs.temporal.io/workflows) define the overall flow of your business process.
Conceptually, a Workflow is a sequence of steps written in your programming language. Workflows orchestrate [Activities](https://docs.temporal.io/activities), which is how you interact with the outside world in Temporal.
You use Activities to make API requests, access the filesystem, or perform other non-deterministic operations.